### PR TITLE
[ECO-1502] Add Linear checkbox item to PR template

### PR DESCRIPTION
**Pull request by alnoki**
# Description

Update the PR template to include a Linear checkbox item

# Testing

The item was appended in this PR's description template upon creation

# Checklist

- [x] Did you update relevant documentation?
- [x] ~Did you add tests to cover new code or a fixed issue?~
- [x] ~Did you update the changelog?~
- [x] ~Did you check off all checkboxes from the linked Linear task?~

### CODE REVIEW

**Review by xbtmatt — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ PR.
 - [ ] Did you update relevant documentation?
 - [ ] Did you add tests to cover new code or a fixed issue?
 - [ ] Did you update the changelog?
+- [ ] Did you check off all checkboxes from the linked Linear task?
```
Comment (xbtmatt) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe a note about not needing to do this if you don't have linear access? Or do external devs have access to the task checkboxes somehow

Comment (alnoki) [reply .github/PULL_REQUEST_TEMPLATE.md]: Addressed in 636090a

**Review by xbtmatt — APPROVED** [review]

**Review by xbtmatt — DISMISSED** [review]
Think you just need to run the readme formatter and it's good

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ PR.
 - [ ] Did you update relevant documentation?
 - [ ] Did you add tests to cover new code or a fixed issue?
 - [ ] Did you update the changelog?
+- [ ] Did you check off all checkboxes from the linked Linear task? (Ignore if
+      you are not a member of Econia Labs)
```
Comment (xbtmatt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
you are not a member of Econia Labs)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ PR.
 - [ ] Did you update relevant documentation?
 - [ ] Did you add tests to cover new code or a fixed issue?
 - [ ] Did you update the changelog?
+- [ ] Did you check off all checkboxes from the linked Linear task? (Ignore if
+you are not a member of Econia Labs)
```
Comment (xbtmatt) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
  you are not a member of Econia Labs)
```

**Review by xbtmatt — APPROVED** [review]

**Pull request merged**